lighthouse-core/runner.js

```diff
@@ -202,6 +210,7 @@ class Runner {
     // Fill remaining audit result fields.
     }).then(auditResult => Audit.generateAuditResult(audit, auditResult))
     .catch(err => {
+      log.warn(audit.meta.name, `Caught exception: ${err.message}`);
```

Comment:
ya true. i saw that earlier. seems okay though. better chance we'll see it since the we've emitted 80 lines of "Evaluating..." since the gatherers logged out. :)